src/main/java/frc/robot/subsystems/Climbers.java

```diff
@@ -70,7 +70,7 @@ private void encoderReset(CANEncoder encoder) {
     encoder.setPosition(0);
   }
 
-  private void climb(double speed) {
+  public void climb(double speed) {
     m_climberMotors.set(speed); // The motor goes at a speed given to it. Not a specific speed
   }
```

Comment:
Don't change this to public. Instead, use one of the public climb methods instead.